tests/unit/src/test/scala/tests/RenameLspSuite.scala

```diff
@@ -1,6 +1,21 @@
 package tests
 
-class RenameLspSuite extends BaseRenameLspSuite("rename") {
+import scala.meta.internal.metals.InitializationOptions
+
+abstract class RenameLspSuite(
```

Comment:
RenameLsp suite take a lot of time, so this would be really bad for out CI to run it twice.